hack/update-deps.sh

```diff
@@ -6,4 +6,8 @@ set -o pipefail
 
 cd "$(dirname "${BASH_SOURCE}")/.."
 
-glide update --strip-vendor
+# https://github.com/golang/go/wiki/Modules#how-to-upgrade-and-downgrade-dependencies
+#
+# Updating all direct and indirect dependencies to latest minor or patch upgrades, where pre-releases
+# are ignored. To list possible updates before installing, execute: "$ go list -u -m all"
+go get -u=patch
```

Comment:
I'm not as big a fan of this since updating deps can get hairy with go mod. We should probably drop this script.

I'd rather go with the approach of updating as per the wiki:

>  A common mistake is thinking go get -u foo solely gets the latest version of foo. In actuality, the -u in go get -u foo or go get -u foo@latest means to also get the latest versions for all of the direct and indirect dependencies of foo. A common starting point when upgrading foo is instead to do go get foo or go get foo@latest without a -u (and after things are working, consider go get -u=patch foo, go get -u=patch, go get -u foo, or go get -u).

Reply:
Good suggestion on this matter, the best course of action is to handle dependencies one by one.